wagtail/admin/templates/wagtailadmin/workflows/task_chooser/includes/results.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n wagtailadmin_tags %}
+{% if tasks %}
+    {% if searchform.is_searching %}
+        <h2 role="alert">
+        {% blocktrans count counter=tasks.paginator.count %}
+            There is {{ counter }} match
+        {% plural %}
+            There are {{ counter }} matches
+        {% endblocktrans %}
+        </h2>
+    {% else %}
+        <h2>{% trans "Tasks" %}</h2>
+    {% endif %}
+
+    <table class="listing">
+        <thead>
+            <tr class="table-headers">
+                <th>
+                    {% trans "Name" %}
+                </th>
+                {% if searchform.specific_task_model_selected %}
+                    {# TODO #}
+                {% else %}
+                    <th>
+                        {% trans "Task type" %}
+                    </th>
+                {% endif %}
+            </tr>
+        </thead>
+        <tbody>
+            {% for task in tasks %}
+                <tr>
+                    <td class="title">
+                        <div class="title-wrapper"><a href="{% url 'wagtailadmin_workflows:task_chosen' task.id %}" class="task-choice">{{ task.name }}</a></div>
+                    </td>
+                    {% if searchform.specific_task_model_selected %}
+                        {# TODO #}
```

Comment:
Is this done in any of the follow-up PRs?
Can't see it in #6171 or #6172

Reply:
Nope, this hasn't been implemented yet

Reply:
👍